assets/js/components/DeregistrationModal/DeregistrationModal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { EOS_CLEANING_SERVICES } from 'eos-icons-react';
+
+import Modal from '@components/Modal';
+import Button from '@components/Button';
+
+function DeregistrationModal({
+  hostName,
+  isOpen = false,
+  onCleanUp,
+  onClose = () => {},
+}) {
+  return (
+    <Modal
+      title={`Clean up data discovered by agent on host ${hostName}`}
+      open={isOpen}
+      onClose={onClose}
+    >
+      <div className="text-gray-500">
+        This will ignore all the past events collected by the agent instance
```

Comment:
The text looks wrong.
Can you update it with the latest text proposed by @stefanotorresi ?